Lib/subprocess.py

```diff
@@ -288,6 +288,7 @@ def _args_from_interpreter_flags():
         'verbose': 'v',
         'bytes_warning': 'b',
         'quiet': 'q',
+        'safe_path': 'P'
```

Comment:
if subprocess.py is too old, rather than editing it yourself, could you try to update it by following the library update guideline on #4564?